WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationContentRouter.swift

```diff
@@ -85,6 +85,9 @@ struct NotificationContentRouter {
             try coordinator.displayFollowersWithSiteID(range.siteID, expirationTime: expirationFiveMinutes)
         case .user:
             try coordinator.displayStreamWithSiteID(range.siteID)
+        case .scan:
+            try coordinator.displayScanWithSiteID(range.siteID)
+
```

Comment:
❓ Do we need to use a feature flag for this?

Reply:
🤦 I totally forgot to do that, I went back and added the checking if scan is available on the site with the intention of feature flagging it too but nope forgot.. 

Done in: a27feaea7095dcfc9ce89490acbe0749407df79e